sdk/storage/sample/datalake_getting_started.cpp

```diff
@@ -3,11 +3,110 @@
 
 #include "datalake/datalake.hpp"
 #include "samples_common.hpp"
+#include <iostream>
 
 SAMPLE(DataLakeGettingStarted, DataLakeGettingStarted)
 void DataLakeGettingStarted()
 {
+  // Please note that you can always reference test cases for advanced usages.
+
   using namespace Azure::Storage::Files::DataLake;
-  auto client = ServiceClient::CreateFromConnectionString(GetConnectionString());
-  auto response = client.ListFileSystems();
+
+  std::string fileSystemName = "sample-file-system";
+  std::string directoryName = "sample-directory";
+  std::string fileName = "sample-file";
+
+  // Initializing a ServiceClient that can then initialize the FileSystemClient or list file
+  // systems.
+  auto serviceClient = ServiceClient::CreateFromConnectionString(GetConnectionString());
+  // Initializing a FileSystemClient that can then initialize the PathClient, FileClient,
+  // DirectoryClient.
+  auto fileSystemClient
+      = FileSystemClient::CreateFromConnectionString(GetConnectionString(), fileSystemName);
+
+  try
+  {
+    // Create file systems and ignore the already exist error.
+    try
+    {
+      fileSystemClient.Create();
+    }
+    catch (Azure::Storage::StorageError& e)
+    {
+      if (e.ErrorCode != "ContainerAlreadyExists")
+      {
+        throw std::move(e);
```

Comment:
What is this?

Reply:
Cannot `throw e` as it contains a `unique_ptr` disabling the copy constructor.

Reply:
rethrow